src/cli_plugin/install/install.js

```diff
@@ -63,7 +63,10 @@ export async function install(settings, logger) {
 
     assertVersion(settings);
 
-    const targetDir = path.join(settings.pluginDir, kebabCase(settings.plugins[0].id));
+    const folderName = settings.plugins[0].folderName
```

Comment:
Can this condition move into a `targetFolderName()` method in settings (might need to be extended into a class with methods), and use `.targetFolderName()` everywhere it's needed? This would avoid duplication and be more OOO.

Reply:
Moved this to `getTargetFolderName()` function.